### PR TITLE
Component Proposal: Carousel

### PR DESCRIPTION
**Pull request by apolokaklab**
### Proposed Changes
- Add a new compenent for Carousel

[Demo Carousel](https://buefy-dev.netlify.com/documentation/carousel/)

### CODE REVIEW

**Comment by afwn90cj93201nixr2e1re** [comment]
Pause is blinking.
Dont merge it to dev from dev, you should create another branch. For avoid 
изображение this stuff.

**Comment by afwn90cj93201nixr2e1re** [comment]
Pause is blinking on hover.

File: src/components/carousel/Carousel.vue [hunk 1]
```diff
@@ -159,11 +156,11 @@ export default {
     },
     methods: {
         startTimer() {
-            if (!this.autoplay || this.interval <= 0 || this.timer) return
+            if (!this.autoplay || this.timer) return
             this.isPause = false
             this.timer = setInterval(() => {
                 this.next()
-            }, this.interval)
+            }, (this.interval || config.defaultCarouselInterval))
```
Comment (afwn90cj93201nixr2e1re) [inline src/components/carousel/Carousel.vue]: what about situation when interval is 0?

Comment (jtommy) [reply src/components/carousel/Carousel.vue]: @afwn90cj93201nixr2e1re you shouldn't set it to zero

Comment (apolokaklab) [reply src/components/carousel/Carousel.vue]: @afwn90cj93201nixr2e1re Then back to use default config, this also work and then why you use interval 0?

Comment (afwn90cj93201nixr2e1re) [reply src/components/carousel/Carousel.vue]: Why not? immediately change on some event.

Comment (service-paradis) [reply src/components/carousel/Carousel.vue]: Interval of 0 would not make any sense for autoplay.

Comment (afwn90cj93201nixr2e1re) [reply src/components/carousel/Carousel.vue]: I know. Read https://github.com/buefy/buefy/pull/1997#discussion_r348090273 again.

Comment (service-paradis) [reply src/components/carousel/Carousel.vue]: Yeah, I read it and it does not make sense anyway. The interval is for the autoplay feature. Not for some event.

**Comment by jtommy** [comment]
@apolokaklab Thanks a lot!

File: src/components/carousel/Carousel.vue (hunk 1 above)
Comment (afwn90cj93201nixr2e1re) [reply src/components/carousel/Carousel.vue]: Seems like props which are reactive is nothing for you. :/

**Comment by service-paradis** [comment]
Good job @apolokaklab!

**Pull request merged**